main.go

```diff
@@ -16,10 +16,11 @@ import (
 var Options struct {
 	AssistedServiceScheme string `envconfig:"ASSISTED_SERVICE_SCHEME"`
 	AssistedServiceHost   string `envconfig:"ASSISTED_SERVICE_HOST"`
-	Port                  string `envconfig:"PORT" default:"8080"`
+	DataDir               string `envconfig:"DATA_DIR"`
 	HTTPSKeyFile          string `envconfig:"HTTPS_KEY_FILE"`
 	HTTPSCertFile         string `envconfig:"HTTPS_CERT_FILE"`
-	DataDir               string `envconfig:"DATA_DIR"`
+	Port                  string `envconfig:"PORT" default:"8080"`
```

Comment:
Is that the image service port or service port? (I could figure that out from the rest of the code obviously, just saying it might be worth to give this a better name)

Reply:
Can the service port even be specified?

Reply:
This is the port the image service binds to.
The port for assisted service can be specified in the `ASSISTED_SERVICE_HOST` var.

Reply:
Alright. Maybe rename this field to `LISTEN_PORT` and add documentation for `ASSISTED_SERVICE_HOST` that it expects a `HOST:PORT` format

Reply:
I'd prefer not to do that in this PR.

I can do it in a follow up, or you can open that PR if you have the time.

Reply:
Sure will do